src/sailfishui/crash-reporter.qml

```diff
@@ -141,6 +141,17 @@ Page {
                 }
             }
 
+            TextSwitch {
+                //% "Use home partition"
+                text: qsTrId("settings_crash-reporter_use_home_partition")
+                //% "Store core dumps in home partition. This can be useful for"
+                //% "capturing large core dumps or when root partition is low on free space."
```

Comment:
I'm late here but this is missing a space between last and first word. Multi-line //% just concatenates the strings. Often the separating space is at the end of the previous line.